<html>
<head>
   <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js" ></script>
   <script>
       $(function(){ $("head").load("header.html") });
   </script>
</head>
<body>
    <div id="left" class="column"></div>
    <div id="right" class="column">
        <div class="top-right"> <h1>BasePlayer manual</h1></div>
        <div class="bottom content">

<h1 id="varanalysis">Variant Analysis</h1>

<h2 id="filtering">Variant quality Filtering</h2>
  <p>Variant quality filtering is operated in <a href="ui.html#varmanager">Variant Manager</a>.</p>
  <h3 id="basic">Basic filters</h3>
  <p>
  Use SNVs tab of the <a href="ui.html#varmanager">Variant Manager</a> to apply basic filtering for SNVs and indels.
  These values are read from the opened VCF files and can be used interactively with the variant visualization.
  </p>
  <p>
  The basic filters include:
  <ul>
    <li><h4>Min. quality score</h4></li>
    <ul>
      <li>This value corresponds to the QUAL field of the VCF file.</li>
    </ul>
    <li><h4>Min. genotype quality score</h4></li>
    <ul>
      <li>This value corresponds to the GQ field of the VCF file.</li>
    </ul>
    <li><h4>Min. coverage</h4></li>
    <ul>
      <li>This value corresponds to the DP field (or AD) of the VCF file.</li>
    </ul>
    <li><h4>Max. coverage</h4></li>
    <ul>
      <li>This value corresponds to the DP field (or AD) of the VCF file.</li>
    </ul>
    <li><h4>Min/max allelic fraction</h4></li>
    <ul>
      <li>This value corresponds to the AD field of the VCF file.</li>
    </ul>
  </ul>
  </p>
  <h3 id="indel">Indel filtering</h3>
  <p>
  To use separate quality filters for indels, go to Indels tab of <a href="ui.html#varmanager">Variant Manager</a> and check "Use indel specific filters".
  </p>
  <br><br>
  If you are using controls, right click the control track and select "Overlap indels" to exclude all overlapping indels found in the control file.
  <br><br>
  We offer repeat region BED files for GRCh37 in <a target="_blank" href="https://baseplayer.fi/tracks/repeats_GRCh37/">https://baseplayer.fi/tracks/repeats_GRCh37/</a>.
  You can open these files for your project and select "Subtract" from the track options to exclude indels in microsatellite regions.

  <h3 id="advanced">Advanced filters</h3>
  If your VCF files have been created by GATK's HaplotypeCaller, you have additional quality parameters for variant filtering. See this post for more information about the
  parameters and suggested threshold values for quality filtering: <a target="_blank" href="https://gatkforums.broadinstitute.org/gatk/discussion/2806/howto-apply-hard-filters-to-a-call-set">
  https://gatkforums.broadinstitute.org/gatk/discussion/2806/howto-apply-hard-filters-to-a-call-set</a>
  <br><br>
  Access additional filter window by clicking the "Min. quality score:" text at the <a href="ui.html#varmanager">Variant Manager</a> SNVs or Indels tabs.
  <br>

  <br>
  In the picture, suggested values from the above link have been set. Note: FS value have less than "<" symbol changed to more than ">",
  which means that all variants having FS value greater than (60 in this case), is excluded.



<h2 id="familial">Familial</h2>
<h3 id="inheritance">Inheritance patterns</h3>
Download <a target="_blank" href="https://www.nature.com/articles/sdata201625">"Ashkenazim Trio"</a> example data for inheritance analyses (GRCh37) by using links below:<br>
(take both vcf.gz and vcf.gz.tbi files for each sample)<br>
<a target="_blank" href="ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/AshkenazimTrio/HG002_NA24385_son/latest/GRCh37/">Son VCF</a>,
<a target="_blank" href="ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/AshkenazimTrio/HG003_NA24149_father/latest/GRCh37/">Father VCF</a>,
<a target="_blank" href="ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/AshkenazimTrio/HG004_NA24143_mother/latest/GRCh37/">Mother VCF</a><br><br>

for read visualization, the BAM files are available at:<br>
(take both sorted.bam and sorted.bam.bai files)<br>
<a target="_blank" href="ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/data/AshkenazimTrio/HG002_NA24385_son/NIST_Stanford_Illumina_6kb_matepair/bams/">Son BAM</a>,
<a target="_blank" href="ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/data/AshkenazimTrio/HG003_NA24149_father/NIST_Stanford_Illumina_6kb_matepair/bams/">Father BAM</a>,
<a target="_blank" href="ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/data/AshkenazimTrio/HG004_NA24143_mother/NIST_Stanford_Illumina_6kb_matepair/bams/">Mother BAM</a><br>
<br>Find more information about the samples here <a target="_blank" href="https://www.nature.com/articles/sdata201625">https://www.nature.com/articles/sdata201625</a>
<br><br>
You can use compare panel to compare variants between family members or inheritance panel for more specified results (requires at least one sample to be set as an affected).
<br><br>
See the video for example analysis at our <a href="https://www.youtube.com/channel/UCywq-T7W0YPzACyB4LT7Q3g"> Youtube channel</a>: "Familial variant study with BasePlayer"
<br><br>
<b>Suggested tracks for exome analysis:</b><br>
mcap_v1_0_forBP.bed.gz in <a href="https://baseplayer.fi/tracks/">https://baseplayer.fi/tracks/</a><br>
<b>Suggested controls for exome analysis:</b><br>
gnomAD_exomes_ALL_GRCh37.vcf.gz in <a href="https://baseplayer.fi/controls/">https://baseplayer.fi/controls/</a><br>
<br><b>Suggested tracks for whole-genome analysis:</b><br>
Encode_Combined_Gm12878_H1hesc_Helas3_Hepg2_Huvec_K562.bed.gz in <a href="https://baseplayer.fi/tracks/">https://baseplayer.fi/tracks/</a>
<br>
GERP_hg19.bw in <a href="https://baseplayer.fi/tracks/">https://baseplayer.fi/tracks/</a>
<br>
Mappability_1000Genomes_pilot_mask.bed.gz in <a href="https://baseplayer.fi/tracks/">https://baseplayer.fi/tracks/</a>
<br>
TFbinding_sites_SELEX_GRCh37.bed.gz in <a href="https://baseplayer.fi/tracks/">https://baseplayer.fi/tracks/</a><br>
<b>Suggested controls for whole-genome analysis:</b><br>
gnomAD_genomes_ALL_GRCh37.vcf.gz in <a href="https://baseplayer.fi/controls/">https://baseplayer.fi/controls/</a>
<br>

<h2 id="somatic">Somatic</h2>
<p>
  In somatic studies you can use the filters, tracks and comparison panel as in familial cases.
  <br>
  See example video of a whole-genome somatic variant analysis at our <a href="https://www.youtube.com/channel/UCywq-T7W0YPzACyB4LT7Q3g">Youtube channel</a>: "Somatic cluster analysis demonstration with BasePlayer"
</p>
<h1 id="readanalysis">Read analysis</h1>
<p>
Read analysis requires aligned BAM or CRAM files with index.
<h2 id="validation">Variant validation</h2>
<p>
Variant quality metrics in VCF files are assessed by the used variant caller. Although the false positive rates of variant
calls have decreased as methods have improved, read level inspection of variant calls and flanking regions are required in
most cases. The integrated variant analysis and visualization in BasePlayer enables rapid workflow for variant validation
 - double-clicking the specific sample in the result table zooms in to the variant locus and expands the sample track. Reads
 appear on the sample track. Read  level zoom enables user to inspect qualities, orientation, and mismatch rates of the
 overlapping and surrounding reads. In  addition, the reference sequence and annotation tracks, such as repeat masker and
 mappability track, provide additional information about the variant calls.
</p>
<a href="images/cohinspect.png"><img src="images/cohinspect.png" ></a>
<h2 id="svs">Structural variant analysis</h2>
<p>
Structural variants break points can be analyzed by inspecting discordant and splitted reads at the break point locus. BasePlayer
groups discordant reads at the bottom to facilitate the analysis. The colors of the reads corresponds to either different kind of
structural variant or chromosome color of the discordant read pair (in case read pairs have been mapped in different chromosomes).
Concordantly mapped read pairs are colored as light green and red (forward and reverse strand).
</p>
<a href="images/structural.png"><img src="images/structural.png" ></a>

<p>
On the above image, there is a LINE1 element at the left side of the left panel. Reads appear as dark, as the mapping quality is lower than set in the settings.
Colorful reads at adjacent to dark reads represents discordant reads, whose pair is aligned to different chromosome (color is determined by the pair's chromosome).
Double click discordant read to split the screen between the different chromosomes. You can split the screen as many times as
the monitor width allows. <br><br> At the right panel, there is a visible break point (mismatches in reads and in the coverage track).
</p>
<p>
Click read to view more information about the read.
</p>
<h2 id="rna">RNA data</h2>
To be added...
<h2 id="long">Long read analysis (3rd generation sequencing)</h2>
<p>
Thousands of base pairs long reads are non-trivial to visualize, as a single read can span multiple genomic breakpoints,
or multiple exons in the case of RNA data. BasePlayer recognizes long-read data, and makes a reasonable effort to present
it properly. For instance, the researcher can divide the main screen between distinct genomic loci by double clicking a
read that has been mapped to multiple locations (figure below). A demonstration video on how to visualize long-read data in
BasePlayer can be viewed at the BasePlayer <a href="https://www.youtube.com/channel/UCywq-T7W0YPzACyB4LT7Q3g">Youtube channel</a>:
"Third-generation sequencing data visualization with BasePlayer"
</p>
<a href="images/Nanopore.png"><img src="images/Nanopore.png" ></a>
<p>
Three split views of the long read are shown,
tracking the split mappings for a single long read.
An inset info panel shows information on the selected read and a schematic illustration of
split read orientations.
</p>
<h2 id="tfbinding">Transcription Factor binding sites</h2>
<p>
BasePlayer has a built in set of transcription factor position specific scoring matrices (PSSMs), obtained from Jaspar database and
<a href="https://www.nature.com/articles/nature15518">https://www.nature.com/articles/nature15518</a>.
If you are analysin the data with GRCh37, you can download aligned motifs from <a href="https://baseplayer.fi/tracks/">https://baseplayer.fi/tracks/</a> (TFbinding_sites_SELEX_GRCh37.bed.gz).
<br><br>
The BED file has the TF name in the "Name" field, which allows BasePlayer to recognize the correct PSSM for correct alignment and visualize it in place at the region track.
<br><br>
<b>Option 1: download aligned motifs from Ensembl BioMart</b><br><br>
Go to <a href="https://www.ensembl.org/">https://www.ensembl.org/</a> (make sure you go to correct version of the reference genome) and click BioMart from the header of the page.
<br>
Choose database -> Ensembl Regulation -> Choose dataset -> Human binding motifs.<br>
Click "Attributes" and check the "Strand" checkbox.<br>
Export the file as compresssed file and leave the TSV selected. Press "GO".<br>
<br>
Open biomart file in BED converter and select "Name" for the Binding matrix column, and
chromosome, start,end and strand as in the original file header. Press convert.
<br>
Uncheck "zero based" from the track settings.
<br>
<br>
<b>Option 2 (requires linux environment, python, bgzip and tabix): align and add motifs manually</b><br><br>
You can add your own motifs to BasePlayer/additions/motifs folder. Create a .txt file and add the desired PSSMs in the file. Use the jaspar_pfm_add.txt as a template.
Motifs will be loaded to memory as the BasePlayer starts.
</p>
<p>
Use for example <a href="https://www.cs.helsinki.fi/group/pssmfind/">MOODS</a> algorithm to align your motifs to the reference genome.
</p>
<p>
Conver MOODS output to BED file with <a href="moods2bed.py">this python script</a>

Use following bash command: <br>
cat moods_output.txt | python moods2bed.py | bgzip -c > output.bed.gz
<br>
<br>
After conversion create the tabix index for the file: <br>
tabix -p bed output_bed.gz
<br>
<br>
The file can now be opened as a BasePlayer track and the motifs are visualized as a sequence logos.
</p>
<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>

</div>
</div>
<script>
$(document).ready(function() {
    $('#left').load('sidebar.html');
});
</script>
</body>
</html>
