<h2 id="vcf">VCF</h2>
<p>
The Variant Call Format (VCF) specifies the format of a text file used in bioinformatics for storing gene sequence variations.
The format has been developed with the advent of large-scale genotyping and DNA sequencing projects, such as the 1000 Genomes Project.
<p>
Read more at <a href="https://en.wikipedia.org/wiki/Variant_Call_Format">https://en.wikipedia.org/wiki/Variant_Call_Format</a>
<p>
VCF files must be sorted by chromosomal position. If index file is not found, BasePlayer creates it automatically.

<h2 id="bam">SAM/BAM/CRAM</h2>
<p>
Sequence Alignment Map (SAM) is a text-based format for storing biological
sequences aligned to a reference sequence developed by Heng Li. It is widely
used for storing data, such as nucleotide sequences, generated by next generation sequencing technologies.
"The format supports short and long reads (up to 128Mbp) produced by different sequencing platforms and
is used to hold mapped data within the GATK and across the Broad Institute, the Sanger Centre, and
throughout the 1000 Genomes project. Sequence Alignment/Map (SAM) format for alignment of nucleotide
sequences (e.g. sequencing reads) to reference sequence(s). May contain base-call and alignment qualities and other data."
<p>
Read more at <a href="https://en.wikipedia.org/wiki/SAM_(file_format)">https://en.wikipedia.org/wiki/SAM_(file_format)</a>
<p>
BAM/CRAM is the compressed binary representation of SAM.
<p>
BasePlayer requires index for BAM/CRAM files.

<h2 id="bed">BED</h2>
<h2 id="bigwig">BigWig</h2>
