<html>
<head>
   <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js" ></script>
   <script>$(function(){ $("head").load("header.html") });</script>
</head>
<body>
    <div id="left" class="column"></div>
    <div id="right" class="column"><div class="top-right"> <h1>BasePlayer manual</h1></div>
        <div class="bottom content">
          <h2 id="mainwindow">Main window</h2>
          <img src="images/meningioma.png" style="width:100%;">
          <p>
          <b>The main window of BasePlayer displaying three samples, a genomic region track and a population control data track.</b>
          Variant view of chromosome 10. (<b>a</b>) Toolbar contains tools for managing the data and navigating the genome. (<b>b</b>) Genome
          bar visualizes chromosome bands and genes. Reference sequence and gene annotation can be changed from dropdown menus at
          left. Memory usage shows used and allocated memory for BasePlayer. (<b>c</b>) Region tracks are used to exclude and annotate variants.
          In addition, various types of regional or base specific scores can be visualized as histograms and TF binding motifs as sequence logos.
          Pressing the red play button will apply the track - this will exclude any variants outside the region.
          (<b>d</b>) Control tracks can be used to exclude common polymorphisms by setting the allele frequency threshold. (<b>e</b>) Sample tracks visualize VCF
          and BAM files of the sample overlayed. Sample name and statistics for currently visible variants shown on the left. Vertical
          lines represent variant calls in a sample, colored as red, green (SNV or indel in coding region, respectively) and grey (variant
          in non-coding region). Height is relative to the sequencing coverage at the variant locus.
        </p>
          <h2 id="toolbar">Toolbar</h2>
            <a href="images/toolbar.png"><img src="images/toolbar.png" ></a>
            <ol>
              <li>File open, additional tool, variant manager and help menus</li>
              <li>Change chromosome</li>
              <li>Go to previous location (back/forward buttons)</li>
              <li>Search bar. Search by position (e.g. 4:5306600) or search by gene name or ENSG code</li>
              <li>Zoom back to chromosomal level</li>
              <li>Current view region (start and end position of the screen)</li>
              <li>Current view length</li>
            </ol>
            <h3 id="filemenu">File menu</h3>
            <table>
              <tr>
                  <td width="30%" align="left" valign="top"><img src="images/filemenu.png" style="width:100%"></td>
                  <td align="left" valign="top">
                    <ul>
                      <li>Add VCFs
                        <p>
                        Open <a href="filetypes.html#vcf">VCF files</a> (vcf.gz preferrably with tabix index file ".vcf.gz.tbi"). Variants will open to <a href="#sampletrack">sample tracks</a>.
                        If BAM file is located at the same directory and has the same name, it will be opened at the same track.
                        Reads will appear when <a href="#samplezoom">zoomed in</a> (view length less than 60kbp)
                        </p>
                      </li>
                      <li>Add BAMs
                        <p>
                        Open <a href="filetypes.html#bam">BAM and CRAM files</a> (index files required (.bam.bai or .cram.crai)). Reads will open to <a href="#sampletrack">sample tracks</a>.
                        Reads will appear when <a href="#samplezoom">zoomed in</a> (view length less than 60kbp). Coverage will be drawn at the zoom distance set in the <a href="#general">settings</a> (default 1Mbp).
                        </p>
                      </li>
                      <li>Add tracks
                        <p>
                          Open track files (BED, BigWig, BigBed, GFF, TSV, txt)</a>. Read more about track files from the <a href="#annotrack">Region/annotation tracks</a> section
                        </p>
                      </li>
                      <li>Add controls (<span style="color:red;">muuta ohjelmaan pieni kirjain</span>)
                        <p>
                        Open control files (single or multi VCF). Control file will open to <a href="#controltrack">Control track</a>.
                        </p>
                      </li>
                      <li>Add track from URL
                        <p>
                        Open track files from external server. Paste ftp or http address to the URL field (see image below) and press "Fetch".

                      </li>
                      <img src="images/pasteurl.png" style="width:70%">
                      <br>

                    </ul>
                  </td>
              </tr>
            </table>
            <ul>
            <li>Open/save project
              <p>
                Open and save BasePlayer projects (.ses).
              </p>
            </li>
            <li>Genomes
              <p>
                Change and add new genomes (opens the <a href="#genomeselect">Genome selector</a>)
              </p>
            </li>
            <li>Update
              <p>
                Update BasePlayer. The button is visible only when updates are available.
                It is also possible to update BasePlayer manually by downloading BasePlayer.jar from
                <a href="https://baseplayer.fi/update">https://baseplayer.fi/update</a>.
              </p>
            </li>
            <li>Clear data
              <p>
                Clears all opened samples, tracks and controls.
              </p>
            </li>
            <li>Exit
              <p>
              Closes BasePlayer.
              </p>
            </li>
          </ul>
            <br>

            <h3 id="tools">Tools</h3>
              <h4 id="tbrowser">Table browser</h4>
                <a href="images/tablebrowser2.png"><img style="width:100%" src="images/tablebrowser2.png"></a>
                <br>
                <a href="images/tablebrowser.png"><img style="width:100%" src="images/tablebrowser.png"></a>

              <h4 id="average">Coverage calculator</h4>
              <a href="images/coveragecalc.png"><img src="images/coveragecalc.png" ></a>
              <h4 id="caller">Variant caller</h4>
              <a href="images/varcaller.png"><img src="images/varcaller.png" style="width:25%"></a>
              <h4 id="BEDconvert">BED converter</h4>
                <p>
                BED converter is a tool, which allows user to import external annotation data, which is not in standard format (BED, BigWig, GFF etc.), to BasePlayer.
                It converts any tab separated file (containing at least chromosome, start and end position fields) to a sorted, compressed and indexed BED file. The file is then usable as an annotation, filtering
                and visualization track. See demonstration video of the BED converter usage with UCSC data in link below.<br>
                <iframe style="width:50%;height:auto;" src="https://www.youtube.com/embed/nSQX2Ctwhso" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
                <br>

                <p>
                  Open tab separated text file (gzipped or plain text) in BED converter. Several rows of the file appears on the table. Select correct header description for the columns using the dropdown menus on the header (red circle).
                  Some columns may have been selected automatically. You must select at least "Chromosome", "Start" and "End" columns for BED conversion. Optional fields are "Name", "Score" and "Strand".
                  Only selected files are added to processed BED file.
                </p>
                  <br>
                  <a href="images/BEDconvert.png"><img style="width:100%" src="images/BEDconvert.png"></a>

          <h2 id="genomebar">Genome bar</h2>
          <a href="images/genomebar.png"><img src="images/genomebar.png" style="width:100%"></a>
          <p>

          </p>
          <a href="images/chromzoom.png"><img src="images/chromzoom.png" style="width:100%"></a>
          <a href="images/isoforms.png"><img src="images/isoforms.png" style="width:100%"></a>


          <h2 id="annotrack">Region/annotation tracks</h2>
          <a href="images/annotracks.png"><img src="images/annotracks.png" ></a>

          <p>

          </p>
          <a href="images/annotracks_zoom.png"><img src="images/annotracks_zoom.png"></a>

          <h2 id="controltrack">Control tracks</h2>
          <a href="images/controltracks.png"><img src="images/controltracks.png" ></a>

          <p>

          </p>

          <h2 id="sampletrack">Sample tracks</h2>
          <a href="images/sampletracks.png"><img src="images/sampletracks.png"></a>
          <p>

          </p>
          <a href="images/sampletracks_zoom.png"><img id="samplezoom" src="images/sampletracks_zoom.png" ></a>
          <br>
          <a href="images/cohinspect.png"><img src="images/cohinspect.png" ></a>
          <br>
          <img src="images/sampledata.png" style="width:50%">
          <h2 id="genomeselect">Genome selector</h2>
          <p>
          Genome selector appears on the first start of the BasePlayer. This tool allows you to add new reference genomes and gene annotations.
          Revisit this window by clicking File -> Genomes -> Add new genome.... or by selecting "Add new reference..." from the reference dropdown menu on the left sidebar of the Genome bar.
          </p>
          <br>
          <img src="images/GenomeSelector.png" style="width:70%;">
          <p>
          <b>1.</b> Fetch new reference genomes and gene annotations from Ensembl if available.<br>
          <b>2.</b> Select Ensembl genome.<br>
          <b>3.</b> Download selected Ensembl reference genome and gene annotation. Files will be automatically installed to your genome directory (9.).<br>
          <b>4.</b> Get download links for the reference genome and gene annotation files. If you can not connect to the Ensembl via BasePlayer, download the files using web browser and add the genome manually using the "Add new reference..." in this window (6.) and "Add new annotation..." (5.)<br>
          <b>5.</b> Add new annotation file (GFF3 or GTF) to the reference genome.<br>
          <b>6.</b> Add new reference genome (FASTA).<br>
          <b>7.</b> Remove selected genome or annotation.<br>
          <b>8.</b> Check, whether selected Ensembl genome has newer gene annotation version available. <br>
          <b>9.</b> Your genome directory, where genomes will be installed.<br>
          <br>
          See instruction video, how to add genomes to BasePlayer:<br>
          <iframe style="width:50%;height:auto;" src="https://www.youtube.com/embed/bj3OR-Vi2XQ" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>

          <h2 id="varmanager">Variant Manager</h2>
          <img src="images/varmanager.png" style="width:70%;">
          <p>
          <b>Variant Manager user interface and functions.</b> (<b>a</b>) Panels for variant quality filtering, variant hiding and sample-wise comparison.
          (<b>b</b>) SNV and Indel quality filters. Separate filters for indels can be activated in the "Indels" tab - same filtering is used by default
          for SNVs and indels. (<b>c</b>) Variant annotation panel including gene effect and output writing options. (<b>d</b>) Output table shows a list of
          genes and variants which pass the filters set in the Variant Manager as well as any activated track. This view also includes tables for
          variant statistics and region track annotations. (<b>e</b>) Sample-wise comparison panel. The upper slider sets the minimum number of cases which
          must harbor a variant in the same gene for these variants to be shown. It can thus be used if the sample set consists of cases with a
          common disease, which might not share the same specific variant even though the same gene is mutated (e.g., sporadic cases or somatic
          mutations in cancer). Lower slider compares individual variants. For instance, the value 2/3 means that only variants shared by two or
          three (all) samples are included and visible on the screen. "Window size for variant cluster" sets variant clustering behavior.
          (<b>f</b>) Inheritance pattern selector. (<b>g</b>) Sample Data dialog. Change / set sample name, gender,
          disease state and parents. (<b>h</b>) Variant Annotation options. Select desired functional classes and options for variants, which are included
          in the results. (<b>i</b>) Output selector for variant annotation results. Select the desired output format for the result file.

          </p>
          <h2 id="settings">Settings</h2>
          <h3 id="general">General</h3>
          <img src="images/genopts.png">
          <h3 id="variants">Variants</h3>
          <img src="images/varopts.png">
          <h3 id="reads">Reads</h3>
          <img src="images/readopts.png">
          <h3 id="appearance">Appearance</h3>
          <img src="images/appearance.png">
          <h3 id="proxy">Proxy</h3>
          <p>
          Set up your proxy settings. Check "Use proxy", select the protocol, host address and port. Finally press "Save".
          <br>
          <img src="images/Proxy.png" style="width:70%;"/>
          <br>
          See instruction video at <br><iframe style="width:50%;height:auto;" src="https://www.youtube.com/embed/vChOTdS9jWQ" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
          </p>

        </div>
        </div>
        <script>

        $(document).ready(function() {
          $('#left').load('sidebar.html');

        });

        </script>
        </body>
        </html>
