<html>
<head>
   <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js" ></script>
   <script>$(function(){ $("head").load("header.html") });</script>
</head>
<body>
    <div id="left" class="column"></div>
    <div id="right" class="column"><div class="top-right"> <h1>BasePlayer manual</h1></div>
        <div class="bottom content">
          <h2 id="mainwindow">Main window</h2>
          <img src="images/meningioma.png" style="width:100%;">
          <p>
          <b>The main window of BasePlayer displaying three samples, a genomic region track and a population control data track.</b>
          Variant view of chromosome 10. (<b>a</b>) Toolbar contains tools for managing the data and navigating the genome. (<b>b</b>) Genome
          bar visualizes chromosome bands and genes. Reference sequence and gene annotation can be changed from dropdown menus at
          left. Memory usage shows used and allocated memory for BasePlayer. (<b>c</b>) Region tracks are used to exclude and annotate variants.
          In addition, various types of regional or base specific scores can be visualized as histograms and TF binding motifs as sequence logos.
          Pressing the red play button will apply the track - this will exclude any variants outside the region.
          (<b>d</b>) Control tracks can be used to exclude common polymorphisms by setting the allele frequency threshold. (<b>e</b>) Sample tracks visualize VCF
          and BAM files of the sample overlayed. Sample name and statistics for currently visible variants shown on the left. Vertical
          lines represent variant calls in a sample, colored as red, green (SNV or indel in coding region, respectively) and grey (variant
          in non-coding region). Height is relative to the sequencing coverage at the variant locus.
        </p>
          <h2 id="toolbar">Toolbar</h2>
            <a href="images/toolbar.png"><img src="images/toolbar.png" ></a>
            <ol>
              <li>File open, additional tool, variant manager and help menus</li>
              <li>Change chromosome</li>
              <li>Go to previous location (back/forward buttons)</li>
              <li>Search bar. Search by position (e.g. 4:5306600) or search by gene name or ENSG code</li>
              <li>Zoom back to chromosomal level</li>
              <li>Current view region (start and end position of the screen)</li>
              <li>Current view length</li>
            </ol>
            <h3 id="filemenu">File menu</h3>
            <table>
              <tr>
                  <td width="30%" align="left" valign="top"><img src="images/filemenu.png" style="width:100%"></td>
                  <td align="left" valign="top">
                    <ul>
                      <li>Add VCFs
                        <p>
                        Open <a href="filetypes.html#vcf">VCF files</a> (vcf.gz preferrably with tabix index file ".vcf.gz.tbi"). Variants will open to <a href="#sampletrack">sample tracks</a>.
                        If BAM file is located at the same directory and has the same name, it will be opened at the same track.
                        Reads will appear when <a href="#samplezoom">zoomed in</a> (view length less than 60kbp)
                        </p>
                      </li>
                      <li>Add BAMs
                        <p>
                        Open <a href="filetypes.html#bam">BAM and CRAM files</a> (index files required (.bam.bai or .cram.crai)). Reads will open to <a href="#sampletrack">sample tracks</a>.
                        Reads will appear when <a href="#samplezoom">zoomed in</a> (view length less than 60kbp). Coverage will be drawn at the zoom distance set in the <a href="#general">settings</a> (default 1Mbp).
                        </p>
                      </li>
                      <li>Add tracks
                        <p>
                          Open track files (BED, BigWig, BigBed, GFF, TSV, txt)</a>. Read more about track files from the <a href="#annotrack">Region/annotation tracks</a> section
                        </p>
                      </li>
                      <li>Add controls
                        <p>
                        Open control files (single or multi VCF). Control file will open to <a href="#controltrack">Control track</a>.
                        </p>
                      </li>
                      <li>Add track from URL
                        <p>
                        Open track files from external server. Paste ftp or http address to the URL field (see image below) and press "Fetch".
                      </p>
                      </li>
                      <img src="images/pasteurl.png" style="width:70%">
                      <br>

                    </ul>
                  </td>
              </tr>
            </table>
            <ul>
            <li>Open/save project
              <p>
                Open and save BasePlayer projects (.ses).
              </p>
            </li>
            <li>Genomes
              <p>
                Change and add new genomes (opens the <a href="#genomeselect">Genome selector</a>)
              </p>
            </li>
            <li>Update
              <p>
                Update BasePlayer. The button is visible only when updates are available.
                It is also possible to update BasePlayer manually by downloading BasePlayer.jar from
                <a href="https://baseplayer.fi/update">https://baseplayer.fi/update</a>.
              </p>
            </li>
            <li>Clear data
              <p>
                Clears all opened samples, tracks and controls.
              </p>
            </li>
            <li>Exit
              <p>
              Closes BasePlayer.
              </p>
            </li>
          </ul>
            <br>

            <h3 id="tools">Tools</h3>
              <h4 id="tbrowser">Table browser</h4>
                <p>
                  Table Browser is a tool for fast browsing through the genome. It is particularly useful when validating break points of the structural variants or the quality of variant calls.
                  You can open any tab separated file (gzipped or raw text file) with the chromosomal position information. Double clicking the position will take user
                  to the selected locus and visualize the data.
                </p>
                <a href="images/tablebrowser2.png"><img style="width:100%" src="images/tablebrowser2.png"></a>
                <p>
                  On above example, BasePlayer output is opened for visual validation of candidate variants.
                  User can select from the user interface (dropdown menus at the table header), which column corresponds to which information
                  Options for header values are:
                  <ol>
                    <li>Position</li>
                    Use this selector, when there is chromosome and position on the same column (e.g. BasePlayer output above; 3:24,446,246)
                    <li>Chromosome</li>
                    When chromosome is on its own column, use this selector.
                    <li>Start</li>
                    If the file has a separate column for chromosome and position(s) use this selector for start position (Figure below).
                    <li>End</li>
                    If the file has a separate column for chromosome and regional (start/end) positions use this selector for end position (Figure below).
                    <li>Sample</li>
                    If the file contains positions for multiple samples select this header value for the column containing sample names. BasePlayer will expand
                    the selected sample when the position is double clicked (Figure above).
                    <li>Gene</li>
                    If the file contains gene names, double clicking the "Gene" column will zoom in to the selected gene (Figure above).
                    <li>Editable</li>
                    This column value marks the column as "Editable", which allows user to edit the table values. The last column is automatically set as Editable,
                    where user can write the validation status of the variant, for example.
                  </ol>
                    <a href="images/tablebrowser.png"><img style="width:100%" src="images/tablebrowser.png"></a>
                  In case you have edited the table, use "Write" button to save the changes.<br>
                  See demonstration video of the usage of "Table Browser" at: <br><iframe width="560" height="315" src="https://www.youtube.com/embed/KBfLZtKvdyw" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
                </p>



              <h4 id="average">Coverage calculator</h4>
              <p>
              Use this tool to get coverage information from the opened BAM files. First, open BAM files as sample tracks "File -> Add BAMs".
              Open "Coverage Calculator" and open BED file containing e.g. targeted regions which will be used in coverage calculation.
              If you have only a few regions in the BED file, check the "Small region" checkbox for faster processing.
              Press "Execute" to start the calculation.
              </p>
              <a href="images/coveragecalc.png"><img src="images/coveragecalc.png" ></a>
              Column explanations:
              <ol>
                <li>Sample</li>
                Sample name
                <li>Average coverage</li>
                Average coverage of the BAM file in opened BED file regions
                <li>Average mapping quality</li>
                Average mapping quality of the reads in the BAM file
                <li>Soft clip rate</li>
                Proportion of the reads having soft clips (<a href="https://www.biostars.org/p/119537/">https://www.biostars.org/p/119537/</a>)
                <li>Zero quality rate</li>
                Proportion of reads having a mapping quality of zero
                <li>Covered (%) (Coverage : Percentage)</li>
                Indicates the proportion of covered regions out of all regions at different coverage levels. I.e.
                "5 : 63" means that if the opened region BED file have total length of 100,000bp, 63% of the total region (63,000bp) have
                been covered at least with 5 reads. The last value e.g. "10+ : 48%" means that 48% of the regions have been covered with at least ten overlapping reads.
                <li>Status</li>
                Shows the progress of the coverage calculation
              </ol>
              <h4 id="caller">Variant caller</h4>
              <p>This is the test version of a simple variant caller, which you can use for BAM files in case you do not have VCF files available.</p>
              <a href="images/varcaller.png"><img src="images/varcaller.png" style="width:25%"></a>
              <ol>
              <li>Min. alt-read count</li>
              Minimum read count for the alternative base calling reads / how many reads are required for the variant call
              <li>Min. coverage</li>
              Minimum total coverage of the locus to be used in variant calling
              <li>Min. mapping quality</li>
              Minimum mapping quality of the reads, which are used in variant calling
              <li>Min. base quality</li>
              Minimum base quality of the mismatched base in the read to be used in the calling
              <li>Require both strands</li>
              Base is called only if both forward and reverse strand reads call the same variant
              <li>Require multiple runs</li>
              In case the used BAM file contains reads from different runs, require that the variant is present in all runs (possibly eliminates run specific errors)
              <li>Calc only for selected</li>
              Calculate variants only for the selected sample (click sidebar of the sample track to select a sample)
              <li>Before annotation</li>
              Start the variant calling after the "Annotation" is pressed in the "Variant Manager"
              </ol>
              <h4 id="BEDconvert">BED converter</h4>
                <p>
                BED converter is a tool, which allows user to import external annotation data, which is not in standard format (BED, BigWig, GFF etc.), to BasePlayer.
                It converts any tab separated file (containing at least chromosome, start and end position fields) to a sorted, compressed and indexed BED file. The file is then usable as an annotation, filtering
                and visualization track. See demonstration video of the BED converter usage with UCSC data in link below.<br>
                <iframe style="width:50%;height:auto;" src="https://www.youtube.com/embed/nSQX2Ctwhso" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
                <br>

                <p>
                  Open tab separated text file (gzipped or plain text) in BED converter. Several rows of the file appears on the table. Select correct header description for the columns using the dropdown menus on the header (red circle).
                  Some columns may have been selected automatically. You must select at least "Chromosome", "Start" and "End" columns for BED conversion. Optional fields are "Name", "Score" and "Strand".
                  Only selected files are added to processed BED file.
                </p>
                  <br>
                  <a href="images/BEDconvert.png"><img style="width:100%" src="images/BEDconvert.png"></a>

          <h2 id="genomebar">Genome bar</h2>
          <p>
          The genome bar visualizes the opened reference sequence, cytobands (at least for humans and mice) and genes. At the whole chromosome level zoom,
          genes which have a screen length more than 1 pixel are named and aligned on different (y-coordinate) levels. I.e. the more you zoom, the more
          genes are visible.
          </p>

          <a href="images/genomebar.png"><img src="images/genomebar.png" style="width:100%"></a>
          <p>

          Left Sidebar contains label for the selected chromosome, dropdown menus for selecting the reference genome / gene annotation, and current memory usage out of the total memory allocated for BasePlayer.
          You can use the dropdown menus to add new annotations and reference sequences as well.
          </p>
          <p>
          The current location of the screen is visualized on top of the cytoband with the red border. You can jump easily to the desired region by dragging mouse on top of the cytoband.
          Centromere is indicated as red triangles.
          </p>
          <a href="images/chromzoom.png"><img src="images/chromzoom.png" style="width:100%"></a>
          <p>
          On the figure above, the last exon of the gene APC is shown. Reference sequence and aminoacids are visible at this zoom level. In addition,
          variants of the all opened samples are collapsed into this view (red and green bars) showing also the amino acid change or the indel effects in place.
          <br>
          <br>
          Tip: drag mouse on top of the reference genome to copy the sequence to clipboard. You can then paste (ctrl-v) the sequence to external software.
          <br>
          Tip2: right click the genome bar screen to copy the position to clipboard.
          </p>
          <a href="images/isoforms.png"><img src="images/isoforms.png" style="width:100%"></a>
          <p>
            On the figure above, the DCP2 gene and its isoforms are shown. Right click any exon of the gene to expand the isoforms (if available). By default, longest transcript is shown.
            Click the exon for more information of the gene and exon. click ENSG or ENST code to open Ensembl page for the gene or specific transcript. Click "View in GeneCards" to see the webpage for selected gene.

          </p>

          <h2 id="annotrack">Region/annotation tracks</h2>
          <p>
          The region track visualizes opened region files (BED, BigWig etc.). The track is also used to annotate or filter opened variants.
          Open region files in "File -> Add tracks". The "Play" button appears to the left sidebar when there are opened VCFs on the sample tracks.
          If region file containts values, the "Histogram" button appears on the sidebar. The region height is determined by its value, when the Histogram view is selected. In case of negative and positive values,
          region track is splitted vertically, zero value in the middle, negative values as red. The scale of the view is shown at the bottom left corner of the track.
          </p>
          <a href="images/annotracks.png"><img src="images/annotracks.png" ></a>

          <p>
          If the region track file is larger than set in the <a href="#general">Settings" -> "General" -> "Big file size for tracks (MB)"</a>, the track is visualized,
          when zoomed in closer than set in <a href="#general">"Settings" -> "General" -> "Processing window size (bp)"</a>.
          </p>
          <p>
          Right click the track or left click the cogwheel symbol to open the track settings:
          <ol>
          <li>Intersect</li>
          If "Play button" is pressed i.e. track is applied and intersect is selected, variants outside the regions are excluded / hidden. If not checked,
          no variants are excluded, but are still annotated for the overlapping occurrences.
          <li>Subtract</li>
          If track is applied and subtract is selected, variants inside the regions are excluded / hidden.
          <li>Zero based</li>
          If start and end positions in the file are not reported starting from zero, uncheck this checkbox and the positions will be shifted accordingly.
          <li>Log scale</li>
          Change the histrogram view to log scale
          <li>Auto scale (in histogram view)</li>
          Scale the peak heights by the highest/lowest value on the screen
          <li>Auto collapse (in region view)</li>
          Collapse and expand track regions as zoomed in/out
          <li>Report affinity changes</li>
          If the track file has a name-field recognized as a transcription factor, affinity changes will be reported on the results if this checkbox is selected
          <li>Apply in annotation</li>
          In case the track file is big ("Settings -> General"), it is annotated using the window size set in the settings and may take a while to complete.
          Use this checkbox to apply the track automatically after the "Variant Manager" -> "Annotation" is pressed.
          <li>Value limit</li>
          Use only regions of the track file, which have higher than the set value
          <li>Left vertical slider</li>
          When not in histogram view, use the slider to set the region bar height
          </ol>
          </p>
          <a href="images/annotracks_zoom.png"><img src="images/annotracks_zoom.png"></a>
          <p>
          If the track file is recognized to include transcription factor binding sites by the "Name" column, binding motifs are visualized at the sequence level zoom.
          </p>
          <h2 id="controltrack">Control tracks</h2>
          <p>
          The control track allows user to apply and set options for control files. Open VCFs for this track in "File -> Add controls". The track will appear above the sample tracks.
          Set the allele frequency limit for the control file. When track is applied, variants having allele frequency more than the limit will be excluded / hidden. Set the limit value to "1"
          for only annotation of variants. Right click the control track to select specialized controlling for indels "Overlap indels". Indels are excluded if the control file have
          overlapping indel at that locus. Otherwise exact match is required for the indel to be filtered out.
          </p>
          <a href="images/controltracks.png"><img src="images/controltracks.png" ></a>
          <p>
          Open control file as a sample for the visualization of the file. In upcoming versions, variants will be visualized in the control track when zoomed in.
          </p>

          <h2 id="sampletrack">Sample tracks</h2>
          <p>
          Sample tracks visualize VCF and BAM files. If you have BAM file named after the VCF file and those are placed in the same folder, both files will be visualized on the same track.
          </p>
          <p>
          The left sidebar shows information about the visible variants on the screen:
          <ol>
            <li>Sample name</li>
            Sample name is the name of the VCF or BAM file by default. You can change it in the sample data dialog (see below)
            <li>Variants</li>
            The number of visible variants (SNVs + indels) on the screen and indel count in parentheses
            <li>Number of heterozygotes</li>
            <li>Number of homozygotes</li>
            <li>Transition / Transversion rate</li>
            <li>Filetypes on the track</li>
            If both VCF and BAM/CRAM is colored green, both files are visualized on the track
            <li>Var height by...</li>
            Variant height on the screen indicates the coverage (by default) at the variant locus. You can change the height determinant from the <a href="#variants">"Settings -> Variants"</a>
            The maximum height value of visible variants is shown in parentheses.
          </ol>
          </p>
          <a href="images/sampletracks.png"><img src="images/sampletracks.png"></a>
          <p>
          Every vertical line on the sample track represents a line of the VCF file (variant call). Red and green lines indicate coding SNVs and indels, respectively. Intronic variants are light gray and intergenic dark gray.
          Variants are visible, if they pass all the filters set in the "Variant Manager" and have not been filtered out by applied control or region tracks.
          <br>
          <br>
          On the above screenshot, the bottom track visualizes ClinVar data and is used as an annotation (yellow indicator on the sidebar).
          </p>
          <a href="images/sampletracks_zoom.png"><img id="samplezoom" src="images/sampletracks_zoom.png" ></a>
          <p>
          If you zoom in to the sequence level (image above), the reads are visualized from the corresponding BAM file. At the top of each track is the coverage track,
          which shows position specific coverage and possible mismathes read from the read data.
          The variant call from VCF is shown behind the reads (red line in the image).
          </p>
          <a href="images/cohinspect.png"><img src="images/cohinspect.png" ></a>
          <p>
          To get more information about the individual reads or variants, you can click them at the sequence level zoom (above image).
          Click the variant to view the VCF information popup, which shows the VCF line information as is.<br>
          Click the read one time to view the BAM information popup:
          <ol>
          <li>Read name</li>
          <li>Chromosomal position and strand (in parentheses)</li>
          <li>Mapping quality (phred scale; 60 is typically maximum value)</li>
          <li>Insert size</li>
          Insert (or fragment) size is inferred from the paired end reads. It is the distance between the start position of the first read and the end position of the second read.
          The number is negative for the second read pair.
          <li>Cigar string contains information about the indels and clipped parts of the read</li>
          <li>Read length</li>
          <li>Chromosomal position of the mate (or pair) and its orientation</li>
          <li>Indicator, whether the read is primary alignment.</li>
          </ol>
          See more read visualization features at <a href="analysis.html#long">3rd generation sequencing data analysis</a>
          </p>
          <p>
          Right click the sidebar, or click the sample name on the sidebar to open "Sample data" dialog (image below):
          </p>
          <table>
            <tr>
                <td width="35%" align="left" valign="top"><img id="sampledata" src="images/sampledata.png" style="width:100%"></td>
                <td align="left" valign="top">
                  <ul>
                    <li>Sample name
                      <p>
                        Change the track/sample name for the project
                      </p>
                    </li>
                    <li>Annotation
                      <p>
                        Set the variant track as an annotation track (e.g. ClinVar data VCF). Filters will not apply to annotation track
                      </p>
                    </li>
                    <li>Gender
                      <p>
                        Select gender for the sample. If sample is set to another sample's father or mother - gender is selected automatically
                      </p>
                    </li>
                    <li>Affected
                      <p>
                        Set sample as an affected. This information is used when using inheritance patterns in variant annotation
                      </p>
                    </li>
                    <li>Mother / Father dropdowns
                      <p>
                        Select mother and/or father for the sample from the other opened samples.
                      </p>

                    </li>
                    <li>Color dropdown
                      <p>
                        Select a distinct color for the family members (sidebar color will change)
                      </p>

                    </li>
                    <li>VCF and BAM path
                      <p>
                        Full path to the directory of the opened VCF and BAM files
                      </p>

                    </li>
                  </ul>
                </td>
            </tr>
          </table>

          <h2 id="genomeselect">Genome selector</h2>
          <p>
          Genome selector appears on the first start of the BasePlayer. This tool allows you to add new reference genomes and gene annotations.
          Revisit this window by clicking File -> Genomes -> Add new genome.... or by selecting "Add new reference..." from the reference dropdown menu on the left sidebar of the Genome bar.
          </p>
          <br>
          <img src="images/GenomeSelector.png" style="width:70%;">
          <p>
          <b>1.</b> Fetch new reference genomes and gene annotations from Ensembl if available.<br>
          <b>2.</b> Select Ensembl genome.<br>
          <b>3.</b> Download selected Ensembl reference genome and gene annotation. Files will be automatically installed to your genome directory (9.).<br>
          <b>4.</b> Get download links for the reference genome and gene annotation files. If you can not connect to the Ensembl via BasePlayer, download the files using web browser and add the genome manually using the "Add new reference..." in this window (6.) and "Add new annotation..." (5.)<br>
          <b>5.</b> Add new annotation file (GFF3 or GTF) to the reference genome.<br>
          <b>6.</b> Add new reference genome (FASTA).<br>
          <b>7.</b> Remove selected genome or annotation.<br>
          <b>8.</b> Check, whether selected Ensembl genome has newer gene annotation version available. <br>
          <b>9.</b> Your genome directory, where genomes will be installed.<br>
          <br>
          See instruction video, how to add genomes to BasePlayer:<br>
          <iframe style="width:50%;height:auto;" src="https://www.youtube.com/embed/bj3OR-Vi2XQ" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>

          <h2 id="varmanager">Variant Manager</h2>
          <a href="images/varmanager.png"><img src="images/varmanager.png" style="width:70%;"></a>
          <p>
          <b>Variant Manager user interface and functions.</b>
          <ol>
            <li>Panels for variant quality filtering, variant hiding and sample-wise comparison</li>
            <li>SNV and Indel quality filters. Separate filters for indels can be activated in the "Indels" tab - same filtering is used by default
            for SNVs and indels</li>
            <p>
            <ol>
              <li><b>Min. quality score:</b> filter variants by the "QUAL" field of the VCF file</li>
              <li><b>Min. genotype quality score:</b> filter variants by the GQ field of the VCF file</li>
              <li><b>Min. coverage:</b> filter variants by the total coverage at the variant position</li>
              <li><b>Max. coverage: </b> filter variants if coverage exceeds this value</li>
              <li><b>Min/max allelic fraction:</b> filter variants by the allelic fraction. Green handle for the minimum and red handle for the maximum value. e.g. remove homozygous variants by setting the max value to less than 100%.</li>

            </ol>
            </p>
            <li>Variant annotation panel including gene effect and output writing options</li>
            <li>Panels for annotation results for genes, clusters, statistics and annotation tracks</li>
            <li>Result table shows a list of genes and variants which pass the filters and options set in the Variant Manager as well as any activated track.
            Color red, yellow, green and gray indicates nonsense, missense, synonymous and non-coding variants, respectively.
            <p>
            Click the gene row to expand the variant view. In case multiple samples share the variant, "Multiple" is shown in the Sample column. Click the multiple row to expand the samples.
            </p>
            <p>
            If controls are applied, there is column for the allele frequency in the control data, including odds ratio with the p-value (fisher exact, right tailed). Also
            applied annotation tracks are shown in separate columns.
            </p>
            </li>
          </ol>
          <a href="images/varmanager_indel.png"><img src="images/varmanager_indel.png" style="width:70%;"></a>
          <p>
          Indel tab is identical to the SNV filtering panel with the exception of the "Use indel specific filters" checkbox.
          Select the checkbox to activate the indel specific filters.
          </p>
          <a href="images/varmanager_hide.png"><img src="images/varmanager_hide.png" style="width:70%;"></a>
          <p>
          Hide panel contains checkboxes to hide / filter out variants with the provided characteristics.
          If you have set the filters for your project and do not need to adjust them, you can check the "Freeze filters"
          checkbox for faster loading times and lesser memory consumption. Variants, which do not pass the variant filters, will not be
          read from the VCF file and not loaded to memory.
          </p>
          <a href="images/varmanager_compare.png"><img src="images/varmanager_compare.png" style="width:70%;"></a>
          <p>
          Use this panel to compare variants or mutated genes in multiple samples.<br>
          The upper slider sets the minimum number of cases which
               must harbor a variant in the same gene for these variants to be shown. It can thus be used if the sample set consists of cases with a
               common disease, which might not share the same specific variant even though the same gene is mutated (e.g., sporadic cases or somatic
               mutations in cancer). Lower slider compares individual variants. For instance, the value 2/3 means that only variants shared by two or
               three (all) samples are included and visible on the screen. "Window size for variant cluster" sets variant clustering behavior.
          </p>
          <a href="images/varmanager_inheritance.png"><img src="images/varmanager_inheritance.png" style="width:70%;"></a>
          <p>
          Inheritance pattern selectors.<br>
          If you have set affected samples from the <a href="#sampledata">"Sample data"</a> dialog, you can use inheritance patters when annotating variants.
          </p>
          <table>
            <tr>
                <td width="25%" align="left" valign="top"><img src="images/varmanager_options.png" style="width:100%"></td>
                <td align="left" valign="top">
                  <p>
                  Gene annotation options. These settings affect to the variant annotation with genes.
                  <br>
                  <b>Only for non-synonymous:</b> do not output synonymous variants to the results<br>
                  <b>Only truncs:</b>output only nonsense mutations<br>
                  <b>Show intronic:</b> output intronic variants<br>
                  <b>Show intergenic:</b> output intergenic variants<br>
                  <b>Show UTR:</b> ouput UTR variants (including non-coding genes and RNAs)<br>
                  <b>All chromosomes:</b> annotate automatically all chromosomes. Otherwise only visible variants on the screen are annotated<br>
                  <b>From this chr?:</b> continue annotation from current chromosome till the end<br>
                  <b>Only autosomes?:</b> annotate variants only for chromosomes 1-22 (in humans)<br>
                  <b>Only selected sample?:</b> output only variants of the selected sample<br>
                  <b>Only stats?:</b> do not output anything on the "Gene" result table. Calculate only statistics to the "Stats" table.<br>
                  <b>Window calculation:</b> annotate variants in batches. (use settings to set the window size)<br>
                  <b>Write directly to a file:</b> do not output anything on the result table. Pressing "Annotation" will open save file dialog for the output file.<br>
                  </p>
                </td>
            <tr>
          </table>
          <br>
          <table>
            <tr>
                <td width="25%" align="left" valign="top"><img src="images/varmanager_hardfilters.png" style="width:100%"></td>
                <td align="left" valign="top">
                  <p>
                  If variants have been called e.g. with GATK HaplotypeCaller, click the "Min. quality score" text on the Variant Manager
                  to see the "Hard filters" popup. The suggested parameters can be found at  <a target="_blank" href="https://gatkforums.broadinstitute.org/gatk/discussion/2806/howto-apply-hard-filters-to-a-call-set">
                  https://gatkforums.broadinstitute.org/gatk/discussion/2806/howto-apply-hard-filters-to-a-call-set</a>
                  <br>
                  Press "Apply" to apply the hard filters.
                  </p>
                  <p>
                  Note: FS parameter has "greater than" symbol set before the value.
                  </p>
                </td>
            </tr>
          </table>
          <br>
          <table>
            <tr>
                <td width="25%" align="left" valign="top"><img src="images/varmanager_output.png" style="width:100%"></td>
                <td align="left" valign="top">
                  <p>
                    Set output file format.<br>
                    <b>TSV:</b> creates tab separated output file for Excel or other software. Every variant and sample are in a separate rows.<br>
                    <b>Compact TSV:</b> same as TSV above, but samples have been collapsed on the same row.<br>
                    <b>VCF:</b> creates VCF file from the outputted variants.
                    <b>Oncodrive:</b> creates a tab separated file recognized by the Oncodrive software
                  </p>
                </td>
            </tr>
          </table>
          <br>
          <a href="images/varmanager_stats.png"><img src="images/varmanager_stats.png" style="width:70%;"></a>
          <p>
          Variant statistics table.<br>
          Includes statistics for all visible variants after annotation.<br>
          </p>
          <h2 id="settings">Settings</h2>
          <p>
          Open BasePlayer settings in "Tools -> Settings" or use the cogwheel symbol at the rightmost of the Toolbar
          </p>
          <h3 id="general">General</h3>

          <img src="images/genopts.png" style="width:70%;">
          <ol>
            <li>Coverage draw distance (bp)</li>
            Set the distance, how far the BAM file is visualized as a coverage histogram. Set it for example to 1 billion to view coverages
            at the chromosomal level (high-coverage whole-genome data will be slow with this setting)
            <li>Processing window size (bp)</li>
            When handling a massive dataset without enough available memory, you can calculate and annotate variants in smaller batches (see <a href="#varmanager">"Variant Manager"</a>).
            This value determines the window size for the batch. Also, if track file bigger than set in below field is opened, it will be visible
            at this distance.
            <li>Set the file size for big track files to prevent the visualization of the data at the chromosomal level. If handling e.g. 80GB file it is
              not feasible to open and visualize the track data as a whole
            </li>
          </ol>
          <h3 id="variants">Variants</h3>
          <img src="images/varopts.png" style="width:70%;">
          <p>
            Select the characteristic of the variant, which will determine its height on the screen (default: coverage).<br>
            <b>Coverage:</b> total coverage in variant position<br>
            <b>Allelic Fraction:</b> a fraction of reads calling the alternative base out of total coverage (values between 0 and 1)<br>
            <b>Quality:</b> value based on the <a href="filetypes.html#vcf">QUAL field of the VCF</a><br>
            <b>GQ:</b> Genotype quality: value based on the <a href="filetypes.html#vcf">GQ value of VCF file</a> (not necessarily set)<br>
            <b>Calls:</b> a nuber of reads calling the variant
          </p>
          <h3 id="reads">Reads</h3>
          <img src="images/readopts.png" style="width:70%;">
          <p>
          Select options for read visualization.<br>
          <b>Maximum insert size:</b> the insert size greater than the set value is considered as a deletion and the read is colored green <br>
          <b>Mapping quality:</b> the read with mapping quality below the set value is colored black<br>
          <b>Base quality:</b> mismatched bases on the read, which have base quality below the set value have dimmer color <br>
          <b>Read depth limit:</b> reads are drawn only at the height of the set value <br>
          <b>Show bases in sofclips:</b> BAM file may not have mismatch information for the soft clipped parts of the read. Select this checkbox to show them anyway<br>
          </p>
          <h3 id="appearance">Appearance</h3>

          <img src="images/appearance.png" style="width:70%;">
          <p>
          Change the appearance of BasePlayer.<br>
          <b>Font size:</b> set the preferred font size and weight used in the software<br>
          <b>Background color:</b> set the background lightness of the region, control and sample tracks<br>
          <b>Sidebar color:</b> set the sidebar colors using RGB values<br>
          </p>
          <h3 id="proxy">Proxy</h3>
          <p>
          Set up your proxy settings. Check "Use proxy", select the protocol, host address and port. Finally press "Save".
          <br>
          <img src="images/Proxy.png" style="width:70%;"/>
          <br>
          See instruction video at <br><iframe style="width:50%;height:auto;" src="https://www.youtube.com/embed/vChOTdS9jWQ" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
          </p>

        </div>
        </div>
        <script>

        $(document).ready(function() {
          $('#left').load('sidebar.html');

        });

        </script>
        </body>
        </html>
